### PR TITLE
Add org.freedesktop.Sdk.Extension.dmd

### PR DESCRIPTION
**Pull request by Cogitri**
Needs #1383

### CODE REVIEW

**Comment by Cogitri** [comment]
Do note that this one isn't strictly required for building D apps - one would usually opt for LDC for its better platform support and stronger optimization here. This is mostly meant for development or apps, which need the very latest version of the D stdlib.

**Comment by barthalion** [comment]
bot, build org.freedesktop.Sdk.Extension.dmd

**Comment by Cogitri** [comment]
bot, build org.freedesktop.Sdk.Extension.dmd

**Comment by Cogitri** [comment]
Ah, I guess it'll take a little for the ldc extension to show up in the repos (since that's what dmd is built from here for simplicity).

**Comment by gasinvein** [comment]
bot, build org.freedesktop.Sdk.Extension.dmd

File: flathub.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+    "only-arches": [
+        "x86_64"
```
Comment (gasinvein) [inline flathub.json]: Do you still need to limit target arches?

Comment (Cogitri) [reply flathub.json]: Yes, dmd is only available on x86 and x86_64

**Pull request closed without merging**